include/CppUTestExt/MockCheckedExpectedCall.h

```diff
@@ -36,10 +36,12 @@ class MockCheckedExpectedCall : public MockExpectedCall
 
 public:
     MockCheckedExpectedCall();
+    MockCheckedExpectedCall(unsigned int numCalls);
     virtual ~MockCheckedExpectedCall();
 
     virtual MockExpectedCall& withName(const SimpleString& name) _override;
-    virtual MockExpectedCall& withCallOrder(unsigned int callOrder) _override;
+    virtual MockExpectedCall& withCallOrder(unsigned int callOrder) _override { return withCallOrder(callOrder, callOrder); }
```

Comment:
It's been a long time, and it had to do with the TI Cl2000 compiler. I can't recall what the problem was and, unfortunately, have no longer access to that compiler for testing. The only way to find out would be to look at old commit comments..... I can't completely rule out that it was unexpected runtime behavior, although I am fairly certain it wasn't.